### DISCUSSION TITLE
Automatic feeder mistake

### DISCUSSION THREAD

**Discussion by Dreamdancer100** [arendst/Tasmota#21678] [Support request]
I control a tank with a Raspi and several ESP8266 flashed with Tasmota and control the whole thing with Iobroker and Blockly controls. Now I have a feed dosage of 300 milliseconds per feed time but sometimes the relay_i stays on true. You can set this permanently for Relay_i 5 using a rule that the device can be on for a maximum of 1 second per true state so that not too much food ends up in the tank
The blockly control is already set so that only 300 ms are triggered. Only when communication via mqtt is interrupted does the relay_i remain set to true and that is not allowed. Relay 5 works on D1 GPIO 5
The big problem is I am from Monday June 24th. at a training course far away and the machine has to run until Sunday because I won't be home for a week.
Please anyone can help

**Comment by sfromis** [comment]
Not clear how exactly you see the issue, but if your question can be boiled down to the relay being on for max 300 ms, no matter what, you could set `PulseTime 3`. Then Tasmota will automatically turn the relay off again after 0.3 seconds, each time it is turned on, without you needing to do more, unless you want to turn it off again faster then the dosage time.

**Reply by Dreamdancer100** [reply]
That sounds great, but since I have several relays you would have to address the relay directly. I saw something about SetOption26 5, so only relay 5 should be addressed, so SetOption26 5 PulseTime 3 would that be correct?

**Reply by sfromis** [reply]
`PulseTime` is a separate command, and you can have it per relay, in the same way as you can have the `Power` command per relay.

Using `SetOption26` makes no difference to this, as this option (which you just can have enabled or not) is about messages emitted from Tasmota always including the index number, also for relay 1 (`Power1` / `PulseTime1`). This option is to make automations simpler, in that they can always expect getting an index, instead of having the special case of no index meaning number 1.

**Reply by Dreamdancer100** [reply]
I found the solution PulseTime5 3 thats only for Relay 5 thanks a lot for your help now I'm happy
and the training course can be start
